jboss-kie-kieserver/added/launch/jboss-kie-kieserver-hooks.sh

```diff
@@ -0,0 +1,105 @@
+#!/bin/bash
+
+source "${JBOSS_HOME}/bin/launch/launch-common.sh"
+source "${JBOSS_HOME}/bin/launch/jboss-kie-common.sh"
+source "${JBOSS_HOME}/bin/launch/logging.sh"
+
+# make sure there is a workbench service within the namespace.
+controllerServiceName=${WORKBENCH_SERVICE_NAME/-/_}
+controllerServiceHost=$(find_env "${controllerServiceName^^}_SERVICE_HOST")
+controllerServicePort=$(find_env "${controllerServiceName^^}_SERVICE_PORT_HTTP")
+
+# param
+# ${1} - the config map payload
+update_config_map() {
+    local payload=${1}
+    # only execute the following lines if this container is running on OpenShift
+    if [ -e /var/run/secrets/kubernetes.io/serviceaccount/token ]; then
+        local namespace=$(cat /var/run/secrets/kubernetes.io/serviceaccount/namespace)
+        local token=$(cat /var/run/secrets/kubernetes.io/serviceaccount/token)
+        local response=$(curl -s --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt \
+            -H "Authorization: Bearer $token" \
+            -H 'Accept: application/json' \
+            -H 'Content-Type: application/strategic-merge-patch+json' \
+            -XPATCH \
+            https://${KUBERNETES_SERVICE_HOST:-kubernetes.default.svc}:${KUBERNETES_SERVICE_PORT:-443}/api/v1/namespaces/${namespace}/configmaps/myapp-kieserver -d ${payload} )
+    fi
+}
+
+if [ -n ${WORKBENCH_SERVICE_NAME} -a -n "${KIE_SERVER_ID}" ]; then
```

Comment:
Do we really need this condition? When WORKBENCH_SERVER_NAME is undefined, we still need to update CM. Plus, KIE_SERVER_ID should be always defined.

Reply:
just to make sure, double check :)